pyvista/utilities/helpers.py

```diff
@@ -215,10 +215,15 @@ def point_array(obj, name):
 
     Returns
     -------
-    numpy.ndarray
+    pyvista.pyvista_ndarray or numpy.ndarray
         Wrapped array.
```

Comment:
Should we explain what determines the return type? Assuming we keep this duality.

Same for field and cell arrays below.

Reply:
Always returning pyvista_ndarray.